app-backend/migrations/src_migrations/main/scala/166.scala

```diff
@@ -0,0 +1,17 @@
+import slick.jdbc.PostgresProfile.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M166 {
```

Comment:
It's possible we can bail on this -- I had in my head that I was going to fix the default Landsat 4 / 5 color correction, but the burden there I think includes searching for and fixing other default color corrections, which requires figuring out what they should be, which is a rabbit hole :upside_down_face: